### PR TITLE
Add news post for playtest-20180825.

### PR DESCRIPTION
**Pull request by pchote**
TODO:
- [x] Add final changelog link
- [x] Deploy https://github.com/OpenRA/OpenRA/issues/15502
- [x] Battlefield news
- [x] Release ID.

### CODE REVIEW

File: content/news/playtest-20180825.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+We are today releasing a [new OpenRA playtest](/download) with fixes for varioius issues reported in [playtest-20180729](https://www.openra.net/news/playtest-20180729/). Thank you to everybody who reported bugs or provided other feedback!
```
Comment (reaperrr) [inline content/news/playtest-20180825.md]: various

File: content/news/playtest-20180825.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+Notable changes in playtest-20180925 include:
```
Comment (reaperrr) [inline content/news/playtest-20180825.md]: 20180*8*25

**Comment by reaperrr** [comment]
Looks good to me apart from those typos.

**Comment by pchote** [comment]
Updated and added battlefield news:
screen shot 2018-08-25 at 21 20 07

File: content/gamenews [hunk 3]
```diff
@@ -1,4 +1,4 @@
...
-	Title: Playtest 20180729
+2018-08-25 21-00-00:
+	Title: Playtest 20180825
 	Author: Paul Chote
-	Content: A new playtest is available for community testing.\nThis pre-release build contains major changes and fixes:\n\n  • Player profiles and badges\n  • New lobby features for players and game admins\n  • Changes to kill-bounties, fake and tech structures,\n    and the Chronoshift in Red Alert\n  • Many other community-voted balance changes in\n    Red Alert and Tiberian Dawn\n  • Significant performance improvements\n  • Fixes for common crashes and other gameplay bugs\n\nRead more and download from www.openra.net and\nlet us know what you think!
+	Content: A new playtest is now available with fixes for many of the issues reported in playtest-20180925. Changes include:\n\n  • Improvements to the player account system\n  • Fixed incorrect infantry death screams\n  • Fixed crashes with text input and bots in the lobby\n  • Fixed AI harvester control\n  • Fixed bugs with the RA Chronoshift-return behaviour\n  • Fixed crashes in several campaign missions\n  • Fixed AppImage compatibility with Linux Mint\n\nRead more and download from www.openra.net and\nlet us know what you think!
```
Comment (reaperrr) [inline content/gamenews]: "...issues reported in playtest-20180**729**."

**Comment by pchote** [comment]
Updated and now hopefully ready to merge.

**Comment by reaperrr** [comment]
👍

**Pull request merged**